### PR TITLE
Password fix on sonarcloud

### PR DESCRIPTION
**Pull request by Sunanda-Boorla**
Signed-off-by: Sunanda-Boorla <sboorla@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
To Solve Security Issue in SonarCloud on Hardcoded Password. I have written a function (getPassword()) that returns a password. This password is assigned to another variable.

### :chains: Related Resources
**https://chefio.atlassian.net/jira/software/c/projects/SHIELD/boards/365?modal=detail&selectedIssue=SHIELD-57**

### :+1: Definition of Done
Solved Security Issue in SonarCloud on Hardcoded Password by writing a function.

### :athletic_shoe: How to Build and Test the Change
 go test client.go client_test.go -v

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

**Review by shaik80 — APPROVED** [review]

File: components/automate-deployment/pkg/usermgmt/client/client_test.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func TestUserMgmtClient(t *testing.T) {
 	t.Run("CreateUser", func(t *testing.T) {
 		name := "testname"
 		email := "test@email.com"
-		password := "GottaCatchEmAll"
+		userPass := getPassword()
```
Comment (atultherajput) [inline components/automate-deployment/pkg/usermgmt/client/client_test.go:51]: Why is the variable name changed from password to userPass?

File: components/local-user-service/users/dex/dex_test.go [hunk 2]
```diff
@@ -403,3 +403,7 @@ func (d *dexAPI) ListRefresh(context.Context, *api.ListRefreshReq) (*api.ListRef
 func (d *dexAPI) RevokeRefresh(context.Context, *api.RevokeRefreshReq) (*api.RevokeRefreshResp, error) {
 	return nil, errors.New("not implemented")
 }
+
+func getPassword()string{
+	return "supercalifragilisticexpialidocious"
+}
```
Comment (shaik80) [inline components/local-user-service/users/dex/dex_test.go:409]: add new line here

**Review by shaik80 — CHANGES_REQUESTED** [review]

File: components/local-user-service/users/dex/dex_test.go (hunk 2 above)
Comment (Sunanda-Boorla) [reply components/local-user-service/users/dex/dex_test.go]: Done

**Review by shivangi-singh1 — APPROVED** [review]

**Review by vivekshankar1 — APPROVED** [review]

**Review by atultherajput — APPROVED** [review]
Looks good.

**Review by vivekshankar1 — APPROVED** [review]

**Pull request merged**